storage/tianmu/handler/tianmu_handler.cpp

```diff
@@ -1476,6 +1476,12 @@ enum_alter_inplace_result TianmuHandler::check_if_supported_inplace_alter([[mayb
                                                                            Alter_inplace_info *ha_alter_info) {
   if ((ha_alter_info->handler_flags & ~TIANMU_SUPPORTED_ALTER_ADD_DROP_ORDER) &&
       (ha_alter_info->handler_flags != TIANMU_SUPPORTED_ALTER_COLUMN_NAME)) {
+    // add alter column operation
```

Comment:
Pls, refine these lines. ugly code.  
And test cases are not sufficient, refer to: 
https://dev.mysql.com/doc/internals/en/test-faults-macros.html

Reply:
update comment and add macros